ucsb-cs156-s21/proj-ucsb-courses-search
# pc - update caniuse database

This updates the "browsers list" for the caniuse-lite npm module, as recommended here, to remove the warning that it is out of date:

* <https://github.com/browserslist/browserslist#browsers-data-updating>

To do this again:
* Pull a clean copy of the `main` branch
* Create a new branch called `your-team-update-caniuse` where `your-team` is, e.g. `w22-4pm-1` or whatever your team name is
* `cd javascript`
* Run: `npx browserslist@latest --update-db`
* Run `git status` and the only change should be in `package-lock.json`
* `git add package-lock.json`
* `git commit -m "xy - update caniuse via npx browserslist@latest --update-db"` where `xy` are your initials
* `git push origin branch-name` where `branch-name` is your branch 
* Make a PR and copy the description from this one into it.
